ultradns-rest/src/main/resources/application.properties

```diff
@@ -0,0 +1,7 @@
+# Feign Client configuration
+feign.connect.timeout.millis=10000
+feign.read.timeout.millis=600000
+
+# UltraDNS REST properties
+ultradns.rest.url=https://test-restapi.ultradns.com/v2
```

Comment:
Sorry the URL comment was meant for this place. Change it to PROD URL before final release.